internal/setup/setup_test.go

```diff
@@ -1570,6 +1570,7 @@ var defaultChiselYaml = `
 		ubuntu:
 			version: 22.04
 			components: [main, universe]
+			suites: [jammy]
```

Comment:
It would be nice to have a new test case in `setupTests` which tests the proposed behaviour, i.e. if the chisel.yaml does not contain `archives.<archive>.suites` field, it raises an error.

Reply:
I fixed the broken tests in other packages and added a test case for this